src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/service/PrisonerEventService.kt

```diff
@@ -24,11 +24,9 @@ class PrisonerEventService(
   }
 
   fun handleReceived(event: PrisonerEvent) {
-    if (ADMISSION_REASONS.contains(event.additionalInformation.reason)) {
```

Comment:
Can you also remove ADMISSION_REASONS, it should be unused now.

Reply:
Removed